### PR TITLE
Add temporary rake task to send apology email

### PR DESCRIPTION
**Pull request by benthorner**
Screenshot 2021-01-08 at 16 20 57

### CODE REVIEW

**Comment by benthorner** [comment]
Tested and works as expected on Integration: the number of generated emails is a fraction of the affected number, which is because we only have the affected emails from yesterday in Integration at the moment.

File: spec/lib/tasks/bulk_email_spec.rb [hunk 1]
```diff
@@ -40,4 +40,35 @@
...
+      create(:email, subject: affected_subject, created_at: Time.zone.local(2021, 1, 8, 13, 50))
+      create(:email, subject: "something else", created_at: Time.zone.local(2021, 1, 8, 13, 40))
+
+      expect(SendEmailWorker)
+        .to receive(:perform_async_in_queue)
+        .with(instance_of(String), queue: :send_email_immediate)
+        .once
+
+      expect { Rake::Task["bulk_email:temp_incident_new_subscribers"].invoke }
+        .to change { Email.count }.by(1)
```
Comment (ChrisBAshton) [inline spec/lib/tasks/bulk_email_spec.rb:67]: Why 1, not 2? I'd expect 2 based on https://github.com/alphagov/email-alert-api/pull/1562/files#diff-08e6964a2fb00793bbdeb6bfc868a53eaf60ec65d46676f3538a677d2ffa7d85R52-R54

Comment (benthorner) [reply spec/lib/tasks/bulk_email_spec.rb:67]: It's meant to be 1: we only want to send a single email, irrespective of how many times a given would-be-subscriber tried to subscribe.

**Review by ChrisBAshton — APPROVED** [review]
Thanks for jumping on this so quickly.

**Pull request merged**